src/custom/components/SearchModal/CurrencyList/CurrencyListMod.tsx

```diff
@@ -17,7 +17,7 @@ import { MouseoverTooltip } from 'components/Tooltip'
 import { MenuItem } from 'components/SearchModal/styleds'
 import Loader from 'components/Loader'
 import { isTokenOnList } from 'utils'
-import ImportRow from 'components/SearchModal/ImportRow'
+import ImportRow from 'components/SearchModal/CurrencyList/ImportRow'
```

Comment:
this one is a typo no? shouldn't it be ImportRow from it's own folder?

Reply:
move `ImportRow` to it's own thing like `components/SearchModal/ImportRow` where inside there it is:
```
ImportRow /
------- ImportRowMod.tsx     <---- UniCode and some mods
------- index.tsx
```

Reply:
done in https://github.com/gnosis/cowswap/pull/1005/commits/a93783b0754d912604ef095cbe0d45064e0a53b7